Makefile.toml

```diff
@@ -49,6 +54,12 @@ else
   set_env BUILD_HDFS_BACKEND_CMD ""
 end
 
+if ${is_not_release} and ${is_dynamic_linking}
```

Comment:
Note: `if not ${is_release} and ${is_dynamic_linking}` doesn't work. https://github.com/sagiegurari/duckscript/issues/326